<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QMainWindow" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="settingsWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="chatSettings">
       <attribute name="title">
        <string>Chat</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="streamText">
          <property name="text">
           <string>Stream response</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="streamChoice">
          <item>
           <property name="text">
            <string>Enabled</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Disabled</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="recoverChatText">
          <property name="text">
           <string>Auto recover previous chat:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="recoverChatChoice">
          <item>
           <property name="text">
            <string>Enabled</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Disabled</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="autoSaveChatText">
          <property name="text">
           <string>Auto save chat</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="autoSaveChatChoice">
          <item>
           <property name="text">
            <string>Immediately</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>When chat ends</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Never</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="multiTurnText">
          <property name="text">
           <string>Multi-turn conversation</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="multiTurnChoice">
          <item>
           <property name="text">
            <string>Enabled</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Disabled</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="maxResponseText">
          <property name="text">
           <string>Max response tokens</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="maxResponseEdit"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="tempText">
          <property name="text">
           <string>Temperature</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="tempEdit"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="historyManText">
          <property name="text">
           <string>History chat files management</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="historyManBtn">
          <property name="text">
           <string>Manage</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="apiPlatText">
          <property name="text">
           <string>API platform</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QComboBox" name="apiPlatChoice">
          <item>
           <property name="text">
            <string>DeepSeek</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Xunfei</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Alibaba</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>OpenAI</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Google</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="modelText">
          <property name="text">
           <string>Model</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QComboBox" name="modelChoice">
          <item>
           <property name="text">
            <string>DeepSeek R1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DeepSeek V3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>[Xunfei]DeepSeek R1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>[Xunfei]DeepSeek V3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>GPT-3.5-Turbo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>GPT-4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>gemini-2-flash</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="keyText">
          <property name="text">
           <string>API key</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLineEdit" name="keyEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="appearanceSettings">
       <attribute name="title">
        <string>Appearance</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="themeText">
          <property name="text">
           <string>Theme</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="themeChoice">
          <item>
           <property name="text">
            <string>System</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Darcula</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AMOLED</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>aqua</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ConsoleStyle</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ElegantDark</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ManjaroMix</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MaterialDark</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ubuntu</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="fontText">
          <property name="text">
           <string>Font</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QFontComboBox" name="fontComboBox"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="langText">
          <property name="text">
           <string>Language</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="langChoice">
          <item>
           <property name="text">
            <string>English(US)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>简体中文</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>正體中文</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="advancedSettings">
       <attribute name="title">
        <string>Advanced</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="customClientText">
          <property name="text">
           <string>Custom AI client</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="customeClientBtn">
          <property name="text">
           <string>Custome</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExport"/>
    <addaction name="actionImport_configuration"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExport">
   <property name="text">
    <string>Export...(&amp;E)</string>
   </property>
   <property name="toolTip">
    <string>Export the current configuration</string>
   </property>
  </action>
  <action name="actionImport_configuration">
   <property name="text">
    <string>Import configuration(&amp;I)</string>
   </property>
   <property name="toolTip">
    <string>Import the configuration file saved before</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
